tripleo-quickstart-config/roles/libvirt/setup/overcloud/templates/baremetalvm.xml.j2

```diff
@@ -31,6 +31,9 @@
       <mac address='{{ node_mac_map.get(item.name).get(network.name) }}'/>
       <source bridge='{{ network.bridge }}'/>
       <model type='{{ overcloud_libvirt_nic_model }}'/>
+{% if network.name == 'provisioning' %}
+      <address type='pci' domain='0x0000' bus='0x01' slot='0x03' function='0x0'/>
```

Comment:
Considering bus numbers from zero up, would it be safer to use '0x00' here? I wonder if libvirt can take up the unused lower bus IDs?

Also, `domain` seems to be optional in the config, may be safer to drop it?